core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -2818,4 +2818,9 @@ public String getEncryptCipherIvSpec() {
     public boolean isEnabledNoAggQuery() {
         return Boolean.parseBoolean(getOptional("kylin.query.enable-no-aggregate-query", FALSE));
     }
+
+    public boolean sparkDeployModeIsYarnCluster() {
```

Comment:
The name of a boolean method would be good at the prefix 'isXXX'.
So this method name could be 'isYarnClusterForSparkDeployMode'.